spec/dummy/db/migrate/20231205185433_pending_user_status.rb

```diff
@@ -0,0 +1,8 @@
+class PendingUserStatus < ActiveRecord::Migration[7.0]
+  def change
+    return if UserStatus.none?
+    return if RadConfig.pending_users?
+
+    execute "DELETE FROM user_statuses WHERE name = 'Pending';"
```

Comment:
I'll deal with this manually on each site, I don't think there will be any existing pending users, but if there are, this will blow up